src/NeoSharp.VM.NeoVM/IApplicationEngine.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Neo.VM;
+using NeoSharp.Types;
+
+namespace NeoSharp.VM.NeoVM
+{
+    public interface IApplicationEngine
+    {
```

Comment:
Should implement the same interface between the VM, defined already in NeoSharp.VM.Interop